src/devices/bus/hp_dio/hp98644.h

```diff
@@ -0,0 +1,57 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont
+
+#ifndef MAME_BUS_HPDIO_98644_H
+#define MAME_BUS_HPDIO_98644_H
+
+#pragma once
+
+#include "hp_dio.h"
+#include "machine/ins8250.h"
+
+#define RS232_TAG       "rs232"
+#define INS8250_TAG     "ins8250"
```

Comment:
Please don't pollute the global namespace with stuff like this.  If you put a macro like this in a header, it propagates to anything that includes the header.